python/ray/node.py

```diff
@@ -299,8 +299,8 @@ def merge_resources(env_dict, params_dict):
                 ray_constants.RESOURCES_ENVIRONMENT_VARIABLE)
             if env_string:
                 env_resources = json.loads(env_string)
-                logger.info(
-                    f"Autosaler overriding resources: {env_resources}.")
+                logger.debug(
+                    f"Autoscaler overriding resources: {env_resources}.")
```

Comment:
Yeah, this seems a bit verbose. Good call

Reply:
yep!